framework/src/engine/endpoint/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export * from './chain';
+export * from './consensus';
+export * from './state';
+export * from './system';
+export * from './txpool';
```

Comment:
make sure we don't expose this to user as its internal and they only need to know about how to call these endpoints